dbux-code/src/traceDetailsView/nodes/TraceValueNode.js

```diff
@@ -12,7 +12,6 @@ export default class TraceValueNode extends TraceNode {
   get value() {
     const { trace: { applicationId, traceId } } = this;
     const dp = allApplications.getById(applicationId).dataProvider;
-    dp.util.getDataNodeValueMessage()
```

Comment:
@MiccWan 
I think, this was a mistake by me... But did you test this after changing it?

Reply:
`TraceValueNode` is deprecated, so instead of testing, I've only checked that `dp.util.getDataNodeValueMessage()` does not make any side effect. Not sure if this is enough? Maybe I should just stay away from those deprecated things :P